pytest_fixtures.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--test_db_url',
```

Comment:
if i understand you correctly, i think you can achieve the same effect by specifying partial urls, e.g. `"postgresql://"` for default port on localhost or `"postgresql:///mydb"` (note 3 slashes, i.e. empty host)